src/index.tsx

```diff
@@ -36,12 +36,14 @@ const addSteps = (steps: Array<Step.StepOptions>, tour: Tour) => {
 
     if (buttons) {
       step.buttons = buttons.map((button: ShepherdButtonWithType) => {
-        const { type, classes, text, action } = button;
+        const { action, classes, disabled, label, text, type } = button;
```

Comment:
I think updating dependencies should be for another PR, not this one.

Reply:
Totally get that, but wasn't sure what was causing everything to fail all the time. Not sure it hurts anything to update them.